### PR TITLE
Coc update

### PR DESCRIPTION
**Pull request by alirezazolanvari**
#### Reference Issues/PRs
#399 

Update the code of conduct according to the Contributor Covenant, version 2.1, available [here](https://www.contributor-covenant.org/version/2/1/code_of_conduct.html).

### CODE REVIEW

File: .github/CODE_OF_CONDUCT.md [hunk 1]
```diff
@@ -1,76 +1,132 @@
+
```
Comment (sadrasabouri) [inline .github/CODE_OF_CONDUCT.md:1]: Extra new line here.

File: .github/CODE_OF_CONDUCT.md [hunk 2]
```diff
@@ -1,76 +1,132 @@
...
+  any kind
+* Trolling, insulting or derogatory comments, and personal or political attacks
 * Public or private harassment
-* Publishing others' private information, such as a physical or electronic
- address, without explicit permission
+* Publishing others' private information, such as a physical or email address,
+  without their explicit permission
 * Other conduct which could reasonably be considered inappropriate in a
- professional setting
+  professional setting
```
Comment (sadrasabouri) [inline .github/CODE_OF_CONDUCT.md:38]: Extra space here.

**Review by sadrasabouri — CHANGES_REQUESTED** [review]
Minor changes needed.

**Review by sadrasabouri — APPROVED** [review]

**Review by sepandhaghighi — CHANGES_REQUESTED** [review]
@alirezazolanvari Thanks for your efforts 💯 
Please update the changelog.
Something like :
`CODE_OF_CONDUCT.md` updated

**Review by sepandhaghighi — APPROVED** [review]
LGTM 💯

**Pull request merged**